app/scripts/metamask-controller.js

```diff
@@ -1552,7 +1520,7 @@ export default class MetamaskController extends EventEmitter {
         },
       },
       env: {
-        isAccountSyncingEnabled: isManifestV3,
+        isAccountSyncingEnabled: false, // TODO: undo this once fixed
```

Comment:
Revert this

Reply:
https://github.com/MetaMask/metamask-extension/pull/27847/commits/2b80d2365acf0a561a6f6ad6d8f1e4aacb0e450e